src/components/ButtonPanel/ButtonPanel.jsx

```diff
@@ -7,32 +7,32 @@ import './ButtonPanel.scss';
 const ButtonPanel = () => (
   <div className="button-panel">
     <div className="group">
-      <Button name="AC" />
-      <Button name="+/-" />
-      <Button name="%" />
-      <Button name="÷" />
+      <Button name="AC" color="lightgrey" />
+      <Button name="+/-" color="lightgrey" />
+      <Button name="%" color="lightgrey" />
+      <Button name="/" />
     </div>
     <div className="group">
-      <Button name="7" />
-      <Button name="8" />
-      <Button name="9" />
+      <Button name="7" color="lightgrey" />
+      <Button name="8" color="lightgrey" />
+      <Button name="9" color="lightgrey" />
       <Button name="X" />
```

Comment:
I trust you can do this before merging
Kindly make this big `X` a small one `x`